content/events/2020-portland-oregon/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Portland, Oregon
+date: '2020-09-12'
+location:
+  country: United States
+  city: Portland, Oregon
+  url: http://cnpdx.us/
+  venue: Puppet Offices
+map: 2020-portland-oregon-venue.png
+sched_slug: kcdpdx
+social:
+  twitter: 'https://twitter.com/cnpdx'
+organizers_email: info@cnpdx.us
+local_law_enforcement: 112
```

Comment:
Should be 911, or?

Reply:
This part doesn't show up on the web preview...but we probably need emergency contact numbers on the real webpage.